### ISSUE TITLE
Consider changing integration tests to use the GRPC interface instead of `lncli`

### ISSUE THREAD

**Issue by aakselrod** [bottlepay/lndsigner#21]
As @joostjager pointed out (https://github.com/bottlepay/lndsigner/pull/11#discussion_r1093168062), the GRPC interface to `lnd`'s API is more stable than the CLI interface provided in `lncli`. However, using it would require either copying ever-larger chunks of the API spec as we expand integration tests, or importing them from `lnd` wholesale, which we've been trying to avoid. We should make a decision on this going forward.

**Comment by joostjager** [comment]
There are already a few lnd protos in `/proto`. Doesn't seem a big deal to add a few more?

**Comment by aakselrod** [comment]
The only ones there are ones we serve. The ones used in itests are under `/itest`, and right now it's just the wallet unlocker. If I do add more, I'd like to do it after we've finalized #11 so that I can convert all of the calls we're making at once. I just added some calls to `Lncli("describegraph")` instead of a `Sleep()` to help wait for proper graph sync.

I'd also like to leave the `Lncli` function in there as a rapid prototyping utility for integration tests, which can be used without adding/updating protos for trying out new functionality.
